pkg/filesystem/subvolume.go

```diff
@@ -480,3 +618,19 @@ func getOmapVal(subVol string) (string, string) {
 
 	return omapval, subvolId
 }
+
+// func getSnapOmapVal is used to get the omapval from the given snapshot
+// omapval is of format csi.snap.427774b4-340b-11ed-8d66-0242ac110005
+// which is similar to volume name csi-snap-427774b4-340b-11ed-8d66-0242ac110005
+// hence, replacing 'csi-vol-' to 'csi.snap.'
```

Comment:
do you mean replacing `csi-snap-` to `csi.snap.`?

Reply:
yes